exasol/python_extension_common/deployment/language_container/flavor_base/build_steps.py

```diff
@@ -1,6 +1,4 @@
-from pathlib import Path
-from typing import Dict
-
+# pylint: disable=E0401
```

Comment:
Maybe a [string-based reference](https://stackoverflow.com/a/23542817/23639321) to the pylint issue is more readable? 

`poetry run pylint --list-msgs | grep E0401 -A1` yields 
```
:import-error (E0401): *Unable to import %s*
  Used when pylint has been unable to import a module.
```

So maybe the following should work: `# pylint: disable=import-error`.

Reply:
I put this line in despair. I think we need to understand why pylint is no longer happy with the import (or maybe it was just on my machine). But I guess for now we just want to make it work, so that we can unblock a couple of other projects. I will create a ticket for that.